about.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,400;0,700;1,400;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/58789452ae.js"
+      crossorigin="anonymous">
+    </script>
+    <link rel="stylesheet" href="index.css">
+    <title>EDULEARN CONFERENCE 2023</title>
+</head>
+
+<body>
+<header>
+  <div class="top-nav"> 
+    <a href="#" target="_blank"><i class="fa-brands fa-facebook-f"></i></a>
+    <a href="#" target="_blank"><i class="fa-brands fa-twitter"></i></a>
+    <a href="#">English</a>
+    <a href="#">My Page</a>
+    <a href="#">LogOut</a>
+    
+    </div>
+<div class="menubar">
+   <div class="navbar">
+    <nav>
+        <img src="images/Union.png" alt="menu">
+    </nav>
+    </div>
+    <nav class="navbar1">
+      <img src="images/logo.png"  alt="logo for convocation"> 
+        <ul>
+          <a  href="index.html">Home</a>
+          <a  href="index.html#courses">Courses</a>
+          <a  href="index.html#speakers">Speakers</a>
+          <a  href="index.html#partners">Sponsers</a>
+          <a  href="index.html">News</a>  
+        </ul>
+    </nav>
```

Comment:
- [ ] As stated in the project requirement You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including: Colors,Typography: font face, size and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design) and Layout: composition and space between elements. the pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable. So kindly fix it with the below points .
- For Desktop size the Nav-bar is divided in a top dark bar and a bottom white bar.The top short bar section with social media links, logout, and language links aligned to the right.The taller bottom bar with a logo and a list of page links.The top bar contains a logo to the left and the top bar contains a list of page links aligned to the right Just like the design below. 

![Capture](https://user-images.githubusercontent.com/93598189/219814193-9073a911-70e1-4d1e-a3d8-1d86ece5d485.PNG)


Your Project Nav-bar Look likes this : 

![Capture](https://user-images.githubusercontent.com/93598189/219814053-0ec7096e-2777-4bd1-a4b2-8fdd343ad049.PNG)